views/default/admin.js.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
IMO, we don't need the actual file. If there are any extensions, they will be rendered despite, correct?